qless-java/src/main/java/com/moz/qless/Queue.java

```diff
@@ -23,6 +22,91 @@
 import redis.clients.jedis.Transaction;
 
 public class Queue {
+
+  public static class JobSpec {
```

Comment:
Right. It's early. I do have all the setters.

I actually intended this to explicitly /not/ be final, because I want to be able to describe something like this:

```java
public class ToolJob {

  private static final String KIJI = "kiji";
  private static final String LOCAL = "local";
  private static final String REDUCERS = "reducers";
  private static final String EXTRA_ARGS = "extraArgs";
  private static final String LEVEL = "level";

  public static class JobSpec extends com.moz.qless.JobSpec {

    public JobSpec setKiji(final String kiji) {
      return data(KIJI, kiji);
    }

    public JobSpec setLocal(final boolean local) {
      return data(LOCAL, local);
    }

    ...

  }

}
```